yelp.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 1.9.2'
+  spec.required_ruby_version = '>= 2.0.0'
```

Comment:
I mean, any ruby below 2.0.0 has not been getting security patches for almost a year.

I do plan to submit a few more PRs soon that will probably use 2.0.0 features, fwiw.